core/src/test/java/org/opensearch/sql/expression/datetime/DateTimeFunctionTest.java

```diff
@@ -625,6 +625,29 @@ public void month() {
     assertEquals(integerValue(8), eval(expression));
   }
 
+  @Test
+  public void month_of_year() {
+    when(nullRef.type()).thenReturn(DATE);
+    when(missingRef.type()).thenReturn(DATE);
+    assertEquals(nullValue(), eval(DSL.month_of_year(nullRef)));
+    assertEquals(missingValue(), eval(DSL.month_of_year(missingRef)));
```

Comment:
Can split up tests. At least the `null`/`missing` should be in their own test.

Reply:
Or use `assertAll`

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/174/commits/4446f366a3659d8a1b087e38e913181fc87d9cc2